testsuite/performance/docker-compose/crossdc/docker-compose-base.yml

```diff
@@ -0,0 +1,154 @@
+version: "2.2"
+
+networks:
+    # DC 1
+    dc1_keycloak:
+        ipam:
+            config:
+            - subnet: 10.1.1.0/24
+    # DC 2
+    dc2_keycloak:
+        ipam:
+            config:
+            - subnet: 10.2.1.0/24
+#    # cross-DC
+#    loadbalancing:
+#        ipam:
+#            config:
+#            - subnet: 10.0.2.0/24
+    # cross-DC
+    db_replication:
+        ipam:
+            config:
+            - subnet: 10.0.3.0/24
+    # cross-DC
+    ispn_replication:
+        ipam:
+            config:
+            - subnet: 10.0.4.0/24
+        
+services:
+
+    infinispan_dc1:
+        build: 
+            context: infinispan
+            args:
+                LOCAL_SITE: dc1
+                REMOTE_SITE: dc2
+        image: keycloak_test_infinispan_dc1:${KEYCLOAK_VERSION:-latest}
+        cpuset: ${INFINISPAN_DC1_CPUSET:-1}
+        mem_limit: ${INFINISPAN_MEMLIMIT:-1g}
+        networks:
+            - ispn_replication
+            - dc1_keycloak
+        environment:
+            PUBLIC_SUBNET: 10.1.1.0/24
+            PRIVATE_SUBNET: 10.0.4.0/24
+            MGMT_USER: admin
+            MGMT_USER_PASSWORD: admin
+            TCP_PING_INITIAL_HOSTS: infinispan_dc1[7600]
+            JAVA_OPTS: ${INFINISPAN_JVM_MEMORY:--Xms64m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m -XX:+DisableExplicitGC} -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true
+        ports:
+            - "9991:9990"
+
+    infinispan_dc2:
+        build: 
+            context: infinispan
+            args:
+                LOCAL_SITE: dc2
+                REMOTE_SITE: dc1
+        image: keycloak_test_infinispan_dc2:${KEYCLOAK_VERSION:-latest}
+        depends_on:
+            infinispan_dc1:
+                condition: service_healthy
+        cpuset: ${INFINISPAN_DC2_CPUSET:-1}
+        mem_limit: ${INFINISPAN_MEMLIMIT:-1g}
+        networks:
+            - ispn_replication
+            - dc2_keycloak
+        environment:
+            PUBLIC_SUBNET: 10.2.1.0/24
+            PRIVATE_SUBNET: 10.0.4.0/24
+            MGMT_USER: admin
+            MGMT_USER_PASSWORD: admin
+            TCP_PING_INITIAL_HOSTS: infinispan_dc1[7600],infinispan_dc2[7600]
+            JAVA_OPTS: ${INFINISPAN_JVM_MEMORY:--Xms64m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m -XX:+DisableExplicitGC} -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true
```

Comment:
would 512m be enough? It needs to hold all the cache contents. Better use larger default, maybe 1g

Reply:
Good point. I will increase the default.